server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -842,11 +853,14 @@ public Configuration updateConfiguration(final UpdateCfgCmd cmd) throws InvalidP
             paramCountCheck++;
         }
         if (accountId != null) {
+            Account account = _accountMgr.getAccount(accountId);
+            _accountMgr.checkAccess(caller, null, true, account);
```

Comment:
@davidjumani 
is there any reason of passing sameOwner=true ?

Reply:
Not needed, changed